### PR TITLE
ci: Install kernel with new script.

### PR DESCRIPTION
**Pull request by jcvenegas**
The Kata packaging now provides an script to build a kernel.

Fixes: #531 

/cc Pennyzct.

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by grahamwhaley** [comment]
@Pennyzct

File: .ci/install_kata_kernel.sh [hunk 1]
```diff
@@ -45,52 +45,28 @@ get_packaged_kernel_version() {
...
-	sudo cp -a "$(realpath vmlinux)" "${kata_kernel_dir}/vmlinux.container"
-	popd
-	popd
-
-	cleanup
+	info "Install kernel from sources"
+	pushd ${tmp_dir} >> /dev/null
+	"${tmp_dir}/${repo_name}/kernel/build-kernel.sh" "setup"
+	"${tmp_dir}/${repo_name}/kernel/build-kernel.sh" "build"
+	sudo "${tmp_dir}/${repo_name}/kernel/build-kernel.sh" "install"
```
Comment (jodh-intel) [inline .ci/install_kata_kernel.sh]: Sweeeeeeeet!

**Comment by jodh-intel** [comment]
lgtm

**Comment by chavafg** [comment]
@jcvenegas I am seeing this error in the fedora CI:

```
scripts/kconfig/conf  --oldconfig Kconfig
#
# configuration written to .config
#
Kernel source ready: /tmp/install-kata-wgASNvaAkhV/kata-linux-4.14.51-unknown 
+ /tmp/install-kata-wgASNvaAkhV/packaging/kernel/build-kernel.sh build
ERROR: path to kernel does not exist, use build-kernel.sh setup
+ exit_handler
+ rm -rf /tmp/install-kata-wgASNvaAkhV
Build step 'Execute shell' marked build as failure
```

**Comment by egernst** [comment]
I have seen this in a couple PRs today: 

```
Test Suite Failed
--- FAIL: TestCRISuite (6.87s)
	cri_test.go:122: Failed to run tests in paralllel: exit status 1
FAIL
./hack/test-utils.sh: line 53: 26061 Terminated              keepalive "sudo PATH=${PATH} ${ROOT}/_output/containerd ${CONTAINERD_FLAGS}" ${RESTART_WAIT_PERIOD} &>${report_dir}/containerd.log
make[1]: *** [test-cri] Error 1
make[1]: Leaving directory `/tmp/jenkins/workspace/kata-containers-tests-centos-7-4-PR/go/src/github.com/containerd/cri'
make: *** [cri-containerd] Error 2
Build step 'Execute shell' marked build as failure
```

**Comment by devimc** [comment]
lgtm

**Pull request merged**